optashift-employee-rostering-gwtui/pom.xml

```diff
@@ -110,10 +110,6 @@
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-simple</artifactId>
     </dependency>
-    <dependency>
-      <groupId>org.jboss.errai</groupId>
-      <artifactId>errai-validation</artifactId>
-    </dependency>
```

Comment:
I think we need this; we use validators on client side (try inputting an emptying name for a Skill on SkillListPanel)

Reply:
Yes, we do, but we don't need it twice :) The dependency is also declared about 10 lines above it.